linkerd/app/src/env.rs

```diff
@@ -408,8 +404,7 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
             context: dst_token?.unwrap_or_default(),
             get_suffixes: dst_get_suffixes?
                 .unwrap_or(parse_dns_suffixes(DEFAULT_DESTINATION_GET_SUFFIXES).unwrap()),
-            get_networks: dst_get_networks?
-                .unwrap_or(parse_networks(DEFAULT_DESTINATION_GET_NETWORKS).unwrap()),
+            get_networks: dst_get_networks?.unwrap_or_default(),
```

Comment:
Just for my understanding, `dst_get_networks` is a `Result<Option>`, yeah?  so the `unwrap_or_default` returns `None` when the environment variable is not provided?

Reply:
It's an IndexSet<IpNet> (so it's an empty set)

Reply:
`dst_get_networks` returns a `Result<Option<IndexSet<T>>, Err>`. The `?` unwraps it to `Option<IndexSet<T>>`. The `unwrap_or_default` returns an `IndexSet<T>` if there is one, or the default (empty `IndexSet<T>`)

Reply:
wait no, the `?` unwraps the `Result`.  So the `unwrap_or_default` returns the empty string?

Reply:
:+1: